### ISSUE TITLE
How to get wdith of image or pane after and during zooming?

### ISSUE THREAD

**Issue by taibhse-designs** [tom91136/GestureFX#3]
System.out.println(pane.getCurrentScale()); shows the current scale factor of the image while zooming, 
 
how can I get the actual width as it is zoomed?

**Comment by tom91136** [comment]
You can use an event handler:
```java
gesturePane.addEventFilter(AffineEvent.CHANGED, event -> {
			System.out.println(event.getTransformedDimension());
		});
```
Or, do you want a property binding that you can use directly?

**Comment by taibhse-design** [comment]
I was looking for a way that on startup I could ensure the image always takes up a set width based on screensize for a comic reading app I am working on, I found this solution to work, 

`       //zoom until full width filled
        float zoomFactor = 0f;
         while ((pane.getCurrentScale() * node.getBoundsInLocal().getWidth()) < width)
        {
            zoomFactor += 0.1f;
            pane.zoomTo(zoomFactor, Point2D.ZERO);
        }`

where width is the width of the frame set to full screen. this is done in a createScene() method as part of the initial start of the UI. Would there be a more elegant solution that I may be missing?

**Comment by tom91136** [comment]
I see, there used to be a method called `cover()` but that got lost for some reason. The current workaround is to set an appropriate `FitMode` (i.e  `COVER` or `FIT`) and then zoom to zero:

```java
pane.setFitMode(FitMode.FIT);
Platform.runLater(() -> pane.zoomTo(0, Point2D.ZERO)); // you may not need Platform.runLater
```
This works because those two FitMode limits the minimum scale to some value regardless of `GesturePane.setMinScale`

**Comment by tom91136** [comment]
`cover()` is definitely coming back so I'll keep this issue opened, thanks for reporting.

**Comment by tom91136** [comment]
Resolved with commit f101f98
